### DISCUSSION TITLE
What should we use discussions for? 🤔

### DISCUSSION THREAD

**Discussion by duskvirkus** [CodingTrain/website#2950] [General]
Hello Passengers, 💖

We're trying out this new GitHub feature. Hopefully it will become another great place in The Coding Train's virtual world 🚂🌈💖. As the first discussion I thought we could ask: What what should we use GitHub discussions for? If you have an idea then please share.

In the past we've talked about using this instead of the [Rainbow-Topics repository](https://github.com/CodingTrain/Rainbow-Topics) to collect video ideas. Or to track more broad discussions about the direction of The Coding Train. Any thoughts on these ideas would be appreciated.

While we figure out what we're doing with this feature feel welcome to play around. Open discussions comment on others and try the feature out in any way you like.

**Comment by KobeLiesenborgs** [comment]
I like the idea of a platform that would allow us to collect thread based ideas, for video topics, stream topics etc etc. 
Maybe we can also use it for public discussions about upcoming changes etc?
